packages/block-library/src/columns/editor.scss

```diff
@@ -1,10 +1,3 @@
-@mixin flex-full-height() {
```

Comment:
As I was looking at this, it appeared that this mixin was used only ever a single time.